R/eDISH.R

```diff
@@ -3,15 +3,121 @@
+#' @param baseline_visitn Value of baseline visit number. Used to calculate mDish. Default: \code{1}. 
+#' @param filters An optional data frame of filters ("value_col") and associated metadata ("label"). Default: \code{NULL}.
+#' @param group_cols  An optional vector of names of grouping variables. Default: \code{NULL}.
+#' @param measure_values A list defining the data values from \code{measure_col} for the lab measures 
+#' used in eDish evaluations. Default: \code{list(ALT = 'Aminotransferase, alanine (ALT)', 
+#' AST = 'Aminotransferase, aspartate (AST)', TB = 'Total Bilirubin', ALP = 'Alkaline phosphatase (ALP)')}.
+#' @param x_options Specifies variable options for the x-axis using the key values from \code{measure_values} (e.g. "ALT"). 
+#' When multiple options are specified, a control allowing the user to interactively change the x variable is shown. Default: \code{c("ALT", "AST", "ALP")}.
+#' @param y_options Specifies variable options for the y-axis using the key values from \code{measure_values} (e.g. "TB"). 
+#' When multiple options are specified, a control allowing the user to interactively change the y variable is shown. Default: \code{"TB"}.
+#' @param measure_bounds Sets upper and lower percentiles used for defining outliers in the "Lab Summary Table"
+#' in the participant details section. Default: \code{c(0.01, 0.99)}.
+#' @param visit_window Default visit window used to highlight eDish points where x and y measures occurred within the specified number of days. 
+#' Editable by user after render. Default: \code{30}.
+#' @param r_ratio_filter Specifies whether the R Ratio filter should be shown. R ratio is defined as: 
+#' (ALT value/ULN for ALT) / (ALP value/ULN for ALP). Default: \code{TRUE}.
+#' @param r_ratio_cut Default cut point for R Ratio filter. Ignored when \code{r_ratio_filter = FALSE}. 
+#' User can update this setting via the UI when \code{r_ratio_filter = TRUE}. Default: \code{0}.
+#' @param showTitle Specifies whether the title should be drawn above the controls. Default: \code{TRUE}.
+#' @param warningText Informational text to be displayed near the top of the controls (beneath the title, if any).
+#'  No warning is displayed if \code{warningText = ""}. Default: \code{"Caution: This interactive graphic is 
+#'  not validated. Any clinical recommendations based on this tool should be confirmed using your organizations 
+#'  standard operating procedures."}.
+#'  
 #' @import htmlwidgets
 #'
 #' @export
-eDISH <- function(data) {
+eDISH <- function(data,
+                  id_col = "USUBJID",
+                  value_col = "STRESN",
+                  measure_col = "TEST",
+                  unit_col = "STRESU",
+                  normal_col_low = "STNRLO",
+                  normal_col_high = "STNRHI",
+                  visitn_col = "VISITN",
+                  baseline_visitn = 1,
+                  filters = NULL,
+                  group_cols = NULL,
```

Comment:
I don't see the `group_cols` in the wireframe - I assume the Shiny app will allow users to specify coloring by which variable.